src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -7,7 +7,7 @@
 using System.Management.Automation;
 using System.Text;
 
-[module: SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Scope = "type", Target = "Microsoft.PowerShell.Commands.ByteCollection")]
+[module: SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Scope = "type", Target = "~T:Microsoft.PowerShell.Commands.ByteCollection")]
```

Comment:
Do we really need the suppression?

Reply:
If we enable the rule and remove the suppression we get:

Error	CA1711	Rename type name ByteCollection so that it does not end in 'Collection'	Microsoft.PowerShell.Commands.Utility	src\Microsoft.PowerShell.Commands.Utility\commands\utility\UtilityCommon.cs	99	Active